### DISCUSSION TITLE
Issue when clicking sign up button

### DISCUSSION THREAD

**Discussion by ABucket** [community/community#144421] [GitHub Education]
### Hi everyone,

Hi, Github Education Community,

I am reaching out regarding an issue I encountered while trying to apply for the GitHub Student Developer Pack. When I visit the application page at https://education.github.com/pack, I see a blue message that says, "We're still processing your application. Once this is complete, you will have access to the student developer pack offers." However, I don't recall submitting any recent application for the Student Developer Pack recently.

Additionally, when I try to sign up by clicking the "Sign up for Student Developer Pack" button, I am redirected to https://github.com/pricing. Similarly, when I click on "Your benefits application" from the drop-down menu in my avatar, I am also redirected to the same pricing page.

Could you kindly assist me in understanding why this is happening? I would appreciate any guidance on how I can properly apply for the Student Developer Pack and resolve this issue.

Thank you in advance for your help.

![1](https://github.com/user-attachments/assets/7d8e3633-a85f-4965-8401-279710ac4a13)

**Comment by mastertanzeel** [comment]
Hello @ABucket ,

In order to fix the problem of accessing or reapplying for the GitHub Student Developer Pack, the first step is to clear cache and cookies from your browser as sometimes they lead to redirection to the wrong pages. You can also use a different browser or apply in incognito mode to eliminate browser issues. If nothing of these resolve your issue, then you might be able to check on your application status on GitHub's support portal. in case there is an application that is still pending review. If nothing from above seems to help, then write to the GitHub Education support team and try to present the problem as best you can, including any details you think might be useful.

I hope this will help you to resolve your issue. Please lemme know if you need any help from me.

Thanks
